Content.Server/Explosion/TriggerSystem.cs

```diff
@@ -30,17 +34,39 @@ public TriggerEvent(IEntity triggered, IEntity? user = null)
     [UsedImplicitly]
     public sealed class TriggerSystem : EntitySystem
     {
+        [Dependency] private readonly IGameTiming _gameTiming = default!;
+
         public override void Initialize()
         {
             base.Initialize();
             SubscribeLocalEvent<TriggerOnCollideComponent, StartCollideEvent>(HandleCollide);
+            SubscribeLocalEvent<TriggerOnProximityComponent, StartCollideEvent>(HandleCollide);
 
             SubscribeLocalEvent<DeleteOnTriggerComponent, TriggerEvent>(HandleDeleteTrigger);
             SubscribeLocalEvent<SoundOnTriggerComponent, TriggerEvent>(HandleSoundTrigger);
             SubscribeLocalEvent<ExplodeOnTriggerComponent, TriggerEvent>(HandleExplodeTrigger);
             SubscribeLocalEvent<FlashOnTriggerComponent, TriggerEvent>(HandleFlashTrigger);
 
             SubscribeLocalEvent<ExplosiveComponent, DestructionEventArgs>(HandleDestruction);
+            SubscribeLocalEvent<TriggerOnProximityComponent, ComponentInit>(CheckEnable);
+            SubscribeLocalEvent<TriggerOnProximityComponent, UnanchoredEvent>(HandleAnchor);
+            SubscribeLocalEvent<TriggerOnProximityComponent, AnchoredEvent>(HandleAnchor);
+
+        }
+
+        private void HandleAnchor(EntityUid uid, TriggerOnProximityComponent component, UnanchoredEvent args)
+        {
+            SetProximityFixture(uid, component, component.Enabled && component.Owner.Transform.Anchored);
+        }
+
+        private void HandleAnchor(EntityUid uid, TriggerOnProximityComponent component, AnchoredEvent args)
+        {
+            SetProximityFixture(uid, component, component.Enabled && component.Owner.Transform.Anchored);
+        }
+
+        private void CheckEnable(EntityUid uid, TriggerOnProximityComponent component, ComponentInit args)
+        {
+            component.EnabledVV = component.Enabled;
```

Comment:
this still massively activates me just do
```suggestion
            SetProximityFixture(uid, component, component.Enabled && component.Owner.Transform.Anchored);
```